pkgs/development/ocaml-modules/decoders-jsonaf/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildDunePackage,
+  decoders,
+  jsonaf,
+  containers,
+  ounit2,
+}:
+
+buildDunePackage rec {
+  pname = "decoders-jsonaf";
+
+  # sub-package built separately from the same source
+  inherit (decoders) src version;
+
+  minimalOCamlVersion = "4.10.0";
```

```suggestion
  minimalOCamlVersion = "4.11.0";
```

`jsonaf` is not available in `nixpkgs` for OCaml < 4.11.